src/pages/Account/Account.scss

```diff
@@ -107,7 +117,9 @@ $page-max-width: 1240px;
 
   &.sparkle {
     margin: 0 auto;
-    background-image: url(/assets/images/sparkle-logo.png);
```

Comment:
do we still need that image? if it's not used anywhere else can we remove it as well?
and if it's removed we could rename `Sparkle_logo_white_yellow.png` to `sparkle-logo.png`.

Reply:
`scripts/bootstrap-new-environments.ts` seems to use it, so I won't delete it

Reply:
FYI: That script is only using it as a default image. If we have a different/better image we can use than can update that script to use it.

Reply:
Also, why did we remove this image from here in general out of curiosity?

Reply:
Since the intent of this change seems to be to replace the old logo with the new, it makes sense to delete the old outdated logo and name the new correct logo as @sunny-viktoryia suggested above.

Reply:
Ok- will make a new pr for this